VersionMiner/Services/IActionOutputService.cs

```diff
@@ -0,0 +1,23 @@
+﻿// <copyright file="IActionOutputService.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+using VersionMiner.Exceptions;
+
+namespace VersionMiner.Services;
+
+/// <summary>
+/// Provides the ability to set the github action output.
```

Comment:
Capitalize
```suggestion
/// Provides the ability to set the Github action output.
```

Reply:
Good catch.  I did a search through the entire code base for other instances.  This was the only one.